### DISCUSSION TITLE
(PVE Perl Style cont.) Don't break line if the resulted line would still pass line length limit

### DISCUSSION THREAD

**Discussion by cheese** [perltidy/perltidy#97] [Q&A]
perltidyrc is the same in https://github.com/perltidy/perltidy/discussions/96

What PVE prefers:
```
my $a = {
    a => 1,
    description => "Only list entries where you have 'Datastore.Audit' or 'Datastore.AllocateSpace' permissions on '/storage/<storage>' long long long"
};
```

What I got from perltidy:
```
my $a = {
    a => 1,
    description =>
	"Only list entries where you have 'Datastore.Audit' or 'Datastore.AllocateSpace' permissions on '/storage/<storage>' long long long",
};
```

**Comment by shancock9** [comment]
Perltidy places line breaks to keep as close to the line limit as possible.  Almost any other automated rule can lead to disaster.  For example, if we consider something like this

```
my $a = {
    a => 1,
    description => $str1.$str2.$str3.$str4.$str5,
};
```

where $str1 etc are all say 100 chars each, then an automated version of the PVE style would not permit any breaks so the line would remain over 500 chars long.
